shared/src/main/scala/tastyquery/ast/Symbols.scala

```diff
@@ -39,7 +39,8 @@ object Symbols {
 
   final class RegularSymbol(override val name: Name, override val owner: Symbol | Null) extends Symbol(name, owner)
 
-  abstract class DeclaringSymbol(override val name: Name, override val owner: Symbol | Null) extends Symbol(name, owner) {
+  abstract class DeclaringSymbol(override val name: Name, override val owner: Symbol | Null)
+      extends Symbol(name, owner) {
```

Comment:
Spurious change here.

Reply:
this is caused by Scalafmt, the line goes beyond the limit otherwise